packages/components/popconfirm/src/popconfirm.ts

```diff
@@ -4,24 +4,20 @@ import { buildProps, definePropType, iconPropType } from '@element-plus/utils'
 import { useTooltipContentProps } from '@element-plus/components/tooltip'
 import type { Component, ExtractPropTypes } from 'vue'
 
+const popConfirmButtonTypes = [...buttonTypes, 'text'] as const
```

Comment:
Maybe don't need  add `text`, `buttonTypes` already contain it.

Reply:
No, we did some updates https://github.com/element-plus/element-plus/pull/7515

Reply:
Ok.